settings.gradle

```diff
@@ -1,3 +1,2 @@
-enableFeaturePreview('VERSION_CATALOGS')
```

Comment:
[Removed in v8.0.0-M1](https://github.com/gradle/gradle/commit/29bbb70e78e70d235053ca5e4a38b0f9bcfe4cce)